Create new token description (#282)
sora-xor/sora-docs: src/kxor.md

  - - meta
    - name: keywords
      content: token burn, Kensetsu, XOR, ETH, KEN, TBCD
---

# Kensetsu XOR (KXOR)

Kensetsu XOR (KXOR) is an essential part of the SORA ecosystem, offering a stable and versatile token that supports both users and builders. It serves as a robust financial tool designed to enhance the value of PUSSY tokens for builders and provide a stable asset with diverse swapping options for users, thus fostering the growth and stability of the SORA ecosystem.

KXOR is pegged to the price of XOR through sophisticated price tools, ensuring a consistent and reliable value

## Usage and Swapping

KXOR can be used to swap for ETH in the XOR-ETH pool, also known as the Chameleon pool.

This feature adds significant liquidity and flexibility for users, allowing them to easily convert KXOR into other valuable assets.

## Overcollateralization and Buy-Back Mechanism

When KXOR is minted, it is overcollateralized by 3% to maintain stability and support the ecosystem.

The overcollateralized 3% is distributed as follows:

- 1% goes towards buying back and burning KEN, similar to other Kensetsu assets.

> - 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem.

Reviewer: ```suggestion
- 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem proposed by the community.
```

- 1% is used to buy back and burn TBCD, further stabilizing the token’s value

## Integration with [PUSSY Tokens](/pussy.md)

> Builders can lock their PUSSY tokens in a Kensetsu vault to mint KXOR.

Reviewer: ```suggestion
Builders can lock their tokens in a Kensetsu vault to mint KXOR.
```

> This process ensures that PUSSY tokens are used productively and enhances the overall utility of the tokens within the ecosystem.

Reviewer: ```suggestion
This process ensures that tokens are used productively and enhances the overall utility of the tokens within the ecosystem. The asset used currently is the PUSSY token.
```

---
title: Kensetsu XOR | SORA Docs
head:
  - - meta
    - name: description

>       content: Learn about KENSETSU, Pegged to XOR, swap for ETH in the XOR-ETH pool, buy-back mechanizm, buring KEN, burning TBCD

Reviewer: ```suggestion
      content: Learn about Kensetsu XOR (KXOR), a robust and flexible financial tool, pegged to the price of XOR that supports the growth and stability of the SORA ecosystem.  
```

  - - meta
    - name: keywords
      content: token burn, Kensetsu, XOR, ETH, KEN, TBCD
---

# Kensetsu XOR (KXOR)

Kensetsu XOR (KXOR) is a crucial component of the SORA ecosystem, providing a stable and versatile token for users and builders. Here's everything you need to know about KXOR.

> ## Pegged to XOR

Reviewer: you can remove this subheader and add the next sentence to the intro

KXOR is pegged to the price of XOR through sophisticated price tools, ensuring a consistent and reliable value

## Usage and Swapping

KXOR can be used to swap for ETH in the XOR-ETH pool, also known as the Chameleon pool.

This feature adds significant liquidity and flexibility for users, allowing them to easily convert KXOR into other valuable assets.

## Overcollateralization and Buy-Back Mechanism

When KXOR is minted, it is overcollateralized by 3% to maintain stability and support the ecosystem.

The overcollateralized 3% is distributed as follows:

- 1% goes towards buying back and burning KEN, similar to other Kensetsu assets.

> - 1% is allocated for buying back and burning the Chameleon Token, a new token in the ecosystem.

Reviewer: is this PUSSY?

Author: No

- 1% is used to buy back and burn TBCD, further stabilizing the token’s value

> ## Integration with PUSSY Tokens

Reviewer: Add a hyperlink to the PUSSY page
